Tasks/NuGetInstaller/nugetinstaller.ts

```diff
@@ -35,7 +35,7 @@ var solution = tl.getPathInput('solution', true, false);
 var filesList = nutil.resolveFilterSpec(solution, tl.getVariable('System.DefaultWorkingDirectory') || process.cwd());
 filesList.forEach(solutionFile => {
     if (!tl.stats(solutionFile).isFile()) {
-        throw new Error(tl.loc('NotARegularFile'));
+        throw new Error(tl.loc('NotARegularFile', solutionFile));
```

Comment:
be careful about throwing when not being called from a Q.Promise or in a try/catch block.  If it's just in the root of the script it will cause execution of the script to blow up with an unhandled exception.  The agent just runs `node <yourScript>`